requirements.txt

```diff
@@ -1,5 +1,5 @@
 Flask==1.0.2
-Flask-pyoidc==1.3.0
+Flask-pyoidc>=2.0.0
```

Comment:
Shouldn't use >= here especially on an essential house service. If they put out 3.0.0 and we're not prepared for it, a build will break prod